niceman/distributions/conda.py

```diff
@@ -207,6 +209,10 @@ def _get_conda_pip_package_details(self, env_export, conda_path):
                            "version": pip_info.get("Version"),
                            "installer": "pip"}
                 packages[pip_dep] = details
+                # Record the origin location (if installed from a local source)
+                if "(" in pip_dep:   # We have an origin location
+                    details["origin_location"] = \
```

Comment:
could we get some basic test for this one?  For an integration test I wonder what would be the most lightweight way... may be we need some minimalistic `niceman/tests/files/samplemod` with some very basic setup.py, and then have it `pip install -e `'ed by conda?  
But I guess we don't need that and you could just test this specific method by feeding it different inputs, and then just mocking those few (if more than one) call to execute_command for pip show

Reply:
Done: 2e1320f

Reply:
I isolated the parsing into a static function so that I can better test it ( 9a93c0c )